en/appendices/3-4-migration-guide.rst

```diff
@@ -40,6 +42,10 @@ behavior that may effect your application:
 * ORM\Query results will not typecast aliased columns based on the original
   columns type. For example if you alias ``created`` to ``created_time`` you
   will now get a ``Time`` object back instead of a string.
+* ``AuthComponent`` now uses a query string to store the redirect URL when an
+  unauthenticated user is redirected to the login page. Previously, this redirect
+  was stored in the session. Using the query string allows for better
+  multi-browser experience.
```

Comment:
Multi browser worked before ;) It was the issues with tabs in the same browser that killed everything.